modules/ob-build-args-and-secrets.adoc

```diff
@@ -0,0 +1,9 @@
+// This module is included in the following assembly:
+//
+// builds/installing-sample-build-strategies.adoc
+
+:_content-type: CONCEPT
+[id="build-args-and-secrets_{context}"]
+= Build arguments and secrets
+
+You can also define array parameters, such as `build-args` and `secrets` in your build strategy for use. You must provide the `build-args` value in the `KEY=VALUE` format and use the `secrets` parameter to store confidential data.
```

Comment:
Same comment as above: "To store confidential data, you must provide the `build-args` value in the `KEY=VALUE` format and use the `secrets` parameter."